ecosystem/nft-metadata-crawler/nft-metadata-crawler-parser/src/models.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © Aptos Foundation
+
+use diesel::prelude::*;
+
+#[derive(Clone, Insertable, Queryable, Selectable)]
+#[diesel(table_name = crate::schema::nft_metadata_crawler_uris)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
+pub struct NFTMetadataCrawlerURIs {
+    pub token_uri: String,
+    pub raw_image_uri: Option<String>,
+    pub raw_animation_uri: Option<String>,
+    pub cdn_json_uri: Option<String>,
+    pub cdn_image_uri: Option<String>,
+    pub cdn_animation_uri: Option<String>,
+    pub image_resizer_retry_count: i32,
+    pub json_parser_retry_count: i32,
+    pub last_updated: chrono::NaiveDateTime,
```

Comment:
Better to let postgres set this directly instead of us passing in a timestamp. This has implications on data replication which might or might not be relevant in this case but good practice.

Reply:
It's a little bit more tedious b/c you have to setup a different query struct, but not that bad. 
https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/indexer-grpc/indexer-grpc-parser/src/models/coin_models/coin_infos.rs#L31-L46

Reply:
Can we use this? https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/indexer-grpc/indexer-grpc-parser/migrations/00000000000000_diesel_initial_setup/up.sql#L19